src/ape/utils/misc.py

```diff
@@ -251,6 +251,10 @@ def to_int(value) -> int:
     raise ValueError(f"cannot convert {repr(value)} to int")
 
 
+def to_snake_case(string: str) -> str:
```

Comment:
There's a library with a bunch of well-tested string conversion functions like this

Reply:
EthTester raw responses contain snake case but turns out we can handle that elsewhere